gateway/src/apicast/policy/liquid_context_debug/liquid_context_debug.lua

```diff
@@ -0,0 +1,30 @@
+local context_content = require('context_content')
+local cjson = require('cjson')
+local policy = require('apicast.policy')
+local _M = policy.new('Liquid context debug')
+
+local new = _M.new
+
+function _M.new(config)
+  local self = new(config)
+  return self
+end
+
+function _M.content(_, context)
+  local content = context_content.from(context)
```

Comment:
We also should add the ngx_variable, right?

Reply:
You're right. And to do that I needed to modify `ngx_variable` so it behaves as a context (linked list): https://github.com/3scale/apicast/pull/849/commits/822345df1bb601fd5bf00c1e799af2f045e92cb0

Reply:
Also, the policy takes that into account now: https://github.com/3scale/apicast/pull/849/commits/98d3d8a946656f12e1d0a14b7bbfa72352fba58c